src/components/Message/MessageItems.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+const StyledMessageItems = styled.div`
+  background: #fff;
+  max-width: 400px;
+  div:nth-child(1) {
+    border-top: 0px;
+  }
+  ${props => props.theme.MessageItems.css}
+`;
+
+export class MessageItems extends React.Component {
```

Comment:
This control doesn't do anything. If you rename `StyledMessageItems` to `MessageItems` you'll get the same result (only better performance - smaller react control tree)